packages/next/client/components/layout-router.client.tsx

```diff
@@ -1,36 +1,89 @@
 import React, { useContext, useEffect, useRef } from 'react'
-import type { ChildProp, Segment } from '../../server/app-render'
+import type {
+  ChildProp,
+  //Segment
+} from '../../server/app-render'
 import type { ChildSegmentMap } from '../../shared/lib/app-router-context'
 import type {
   FlightRouterState,
   FlightSegmentPath,
-  FlightDataPath,
+  // FlightDataPath,
 } from '../../server/app-render'
 import {
   LayoutRouterContext,
   GlobalLayoutRouterContext,
 } from '../../shared/lib/app-router-context'
 import { fetchServerResponse } from './app-router.client'
-import { matchSegment } from './match-segments'
+// import { matchSegment } from './match-segments'
 
 /**
  * Check if every segment in array a and b matches
  */
-function equalSegmentPaths(a: Segment[], b: Segment[]) {
-  // Comparing length is a fast path.
-  return a.length === b.length && a.every((val, i) => matchSegment(val, b[i]))
-}
+// function equalSegmentPaths(a: Segment[], b: Segment[]) {
```

Comment:
might wanna clean up the commented methods?

Reply:
These are intentionally commented as we might be able to re-enable the fast path later.